products/mssqlserver.md

```diff
@@ -11,135 +11,408 @@ versionCommand: select @@version
 releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
 
 activeSupportColumn: true
+releaseColumn: true
 releaseDateColumn: true
 extendedSupportColumn: Extended Security Updates
 
+# releaseDate/support/eol/extendedSupport: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
+# Overview over all versions: https://learn.microsoft.com/troubleshoot/sql/releases/download-and-install-latest-updates
 releases:
 -   releaseCycle: "2022"
     codename: Dallas
-    releaseLabel: "2022 CU9"
+    releaseLabel: "2022"
     releaseDate: 2022-11-16
     support: 2028-01-11
     eol: 2033-01-11
     extendedSupport: false
-    latest: "16.0.4085.2"
-    latestReleaseDate: 2023-10-12
-    link: https://learn.microsoft.com/troubleshoot/sql/releases/sqlserver-2022/cumulativeupdate9
-
--   releaseCycle: "2019"
-    codename: Seattle
-    releaseLabel: "2019 CU23"
-    releaseDate: 2019-11-04
-    support: 2025-01-07
-    eol: 2030-01-08
-    extendedSupport: false
-    latest: "15.0.4335.1"
-    latestReleaseDate: 2023-10-12
-    link: https://learn.microsoft.com/troubleshoot/sql/releases/sqlserver-2019/cumulativeupdate23
-
--   releaseCycle: "2017"
-    codename: Helsinki
-    releaseLabel: "2017 CU31"
-    releaseDate: 2017-09-29
-    support: 2022-10-11
-    eol: 2027-10-12
-    extendedSupport: false
-    latest: "14.0.3465.1"
-    latestReleaseDate: 2023-10-10
-    link: https://support.microsoft.com/help/5029376
+    latestGdr: "16.0.1105.1"
+    latestGdrLink: https://support.microsoft.com/help/5029379
+    latest: "16.0.4095.4 CU10"
```

Comment:
Looking at https://learn.microsoft.com/en-us/troubleshoot/sql/releases/download-and-install-latest-updates#sql-server-2022 I think it could be better to track the CU in an individual field. It feels more natural to only have the version in the latest field, at least for API users. WDYT ?

We may miss a _latestReleaseLabel_ feature to display all those informations though ?

Reply:
That's a good point I see three options:
1. Block this PR until we have the feature `latestReleaseLabel`
2. Remove the short terms in latest until we have the feature `latestReleaseLabel`
3. Leave as it is now and update when we have the feature `latestReleaseLabel`

I vote for option 3.

Reply:
Created #4196 for the new feature `latestReleaseLabel`